scripts/electron/preload.js

```diff
@@ -1,5 +1,5 @@
-const {contextBridge, ipcRenderer} = require('electron');
-
+const {contextBridge, ipcRenderer, remote} = require('electron');
+const {getMaximumMemory, setMaximumMemory} = require('../memconfig.js');
```

Comment:
When referencing a module in another project, we prefer to use the full Node path:

```
require('opensphere-electron/app/src/memconfig.js')
```

Reply:
That didn't seem to work, is it because the preload.js is copied into `opensphere-electron` directory?

Reply:
You're right, that won't work in the built application. The current solution works.